packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -123,8 +144,12 @@ class HorizontalOverflow extends Foundation<
         if (canUseDOM() && this.horizontalOverflowItemsRef.current) {
             this.horizontalOverflowItemsRef.current.addEventListener(
                 "scroll",
-                this.throttled
+                this.throttledScroll
             );
+            window.addEventListener("resize", this.throttledResize);
+            if (this.overflow !== this.isOverflow()) {
```

Comment:
This will:
A. Always fire because this.overflow has not been set
B. Does not indicate a change in overflow since no children will have been loaded during the constructor phase

Please remove this if... else block and initially set this.overflow = false.